### PR TITLE
Do not archive mirrored pages

### PR DESCRIPTION
**Pull request by MizukiTemma**
### Short description

This PR solves the inconsistent state of mirrored pages by prevanting archiving of mirrored pages.

### Proposed changes

- Deactivate the archiving icon of mirrored pages in the tree 
- Hide the archiving button in the form if the page is mirrored
- Check if there are mirorring pages before archiving a page (both in single request and bulk action)
- Adjust error/success messages of the bulk action and titles of the icon

### Side effects

- None?
- I have chosen the first one among the two suggested solutions (expected behaviour) because it seems to be more in accordance with the existing behaviour of page deletion and probably causes less confusion than the second option.

### Resolved issues

Fixes: #2171 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by timoludwig — COMMENTED** [review]
Thanks a lot, works as expected! :muscle: 

I thought that maybe it could be helpful to know which pages prevent the page from being archived, so in case they want to proceed they can e.g. contact the manager of the other region to remove the live content etc...

**Comment by MizukiTemma** [comment]
@timoludwig In the action box it shows where the page is mirrored. I'll modify the explanation to "Um diese Seite zu löschen **oder zu archivieren.**....."

Where am I mirrored

**Review by timoludwig — APPROVED** [review]
Cool, thanks! :+1:

**Comment by svenseeberg** [comment]
I'm a little worried that some regions have to maintain pages because other regions do mirror them. Can we think of a solution here? Maybe a button to remove all mirror pages by force from the source region and notify the maintainers via mail?

**Comment by MizukiTemma** [comment]
That's a nice idea 😃 @osmers What way do you think is the best to inform content creators that a mirrored page has been removed if we introduce force archiving/deleting of mirrored pages? I have a bit of worry that not all of them check E-mails...

File: integreat_cms/locale/de/LC_MESSAGES/django.po
Comment (david-venhoff) [inline integreat_cms/locale/de/LC_MESSAGES/django.po:6057]:
```suggestion
"diesen Seiten entfernen:"
```

**Review by david-venhoff — APPROVED** [review]
Looks good to me. As a small suggestion, you might consider to move the `is mirrored` checks into the `page.archive()` method, to reduce code duplication. I don't think this is blocking the pr though.

**Comment by MizukiTemma** [comment]
> As a small suggestion, you might consider to move the `is mirrored` checks into the `page.archive()` method, to reduce code duplication. I don't think this is blocking the pr though.

Nice idea, though I'll leave the code as it is to keep the current check & message schema and to avoid bulk archive action getting inturruped at the first mirrored page 🙂

**Comment by osmers** [comment]
@MizukiTemma good question - I think generally they are registered with their work account and should therefore read the emails. Maybe you can also always send an email to me so that I am aware and can inform the corresponding person from the service team to double check with the municipality. 
I just had the issue as well that I need to archive the corona pages but can't bcs they are mirrored in many regions. 
Did you introduce a solution already or not yet?

**Comment by MizukiTemma** [comment]
@osmers Thank you for feedback! This notification function is not yet implemented. I have updated the issue for that #2338

**Comment by osmers** [comment]
@MizukiTemma at the same time I noticed that archived pages are also shown as mirroring a page and created an issue for that bcs only live pages should be in the list. Otherwise people would get notified that content has been removed even though they already archived the page that had the removed content.

**Pull request merged**
